docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -10,28 +10,60 @@ ms.date: 06/12/2023
 
 The .NET SDK makes it possible to reduce the size of self-contained apps by [trimming](trim-self-contained.md), which removes unused code from the app and its dependencies. Not all code is compatible with trimming, so .NET 6 provides trim analysis warnings to detect patterns that may break trimmed apps. To resolve warnings originating from the app code, see [resolving trim warnings](#resolve-trim-warnings). This article describes how to prepare libraries for trimming with the aid of these warnings, including recommendations for fixing some common cases.
 
-## Enable library trim warnings
+## Prerequisites
+
+### [.NET 6](#tab/net6)
+
+[.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet)
+
+### [.NET 7](#tab/net7)
+
+[.NET 7 SDK](https://dotnet.microsoft.com/download/dotnet)
 
-> [!TIP]
-> Ensure you're using the .NET 6 SDK or later for these steps. They will not work correctly in previous versions.
+### [.NET 8+](#tab/net8plus)
+
+[.NET 8 SDK](https://dotnet.microsoft.com/download/dotnet)
```

Comment:
> Maybe it would be better to include just one download link and remove the 6/7/8 sections? Currently it reads to me like "For .NET 6, the prerequisite is .NET 6", etc.

We've got to get rid of the awful bling for .NET 6. 

<img width="531" alt="image" src="https://github.com/dotnet/docs/assets/3605364/3934a91e-95d7-48b6-b8f2-9bd81d2ba505">

Soon .NET 8 will be released for sure we don't want to wade through the awful mess.

BTW, I'm just getting started on this draft.

All the pivot mess could be avoided if versions were supported like they are in the ASP.NET Core docs.